site/content/en/_index.html

```diff
@@ -46,16 +46,16 @@
 
 {{< blocks/section color="primary"  >}}
 
-{{% blocks/feature icon="fab fa-slack" title="Join us on Slack" url="https://join.slack.com/t/open-match/shared_invite/enQtNDM1NjcxNTY4MTgzLTM5ZWQxNjc1YWI3MzJmN2RiMWJmYWI0ZjFiNzNkZmNkMWQ3YWU5OGVkNzA5Yzc4OGVkOGU5MTc0OTA5ZTA5NDU" %}}
+{{% blocks/feature icon="fab fa-slack" title="Join us on Slack" url="https://join.slack.com/t/open-match/shared_invite/enQtNDM1NjcxNTY4MTgzLTM5ZWQxNjc1YWI3MzJmN2RiMWJmYWI0ZjFiNzNkZmNkMWQ3YWU5OGVkNzA5Yzc4OGVkOGU5MTc0OTA5ZTA5NDU" url_text="Join us" %}}
 Join the community on Slack
 {{% /blocks/feature %}}
 
 
-{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/googleforgames/open-match" %}}
+{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/googleforgames/open-match" url_text="Contributes to Open Match" %}}
```

Comment:
Should be "Contribute to Open Match" (singular)

Reply:
Thanks for pointing it out!